ray-operator/apis/ray/v1/rayservice_types.go

```diff
@@ -80,7 +98,11 @@ type RayServiceSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	// Defines the applications and deployments to deploy, should be a YAML multi-line scalar string.
 	// +optional
-	ServeConfigV2  string         `json:"serveConfigV2,omitempty"`
+	ServeConfigV2 string `json:"serveConfigV2,omitempty"`
+	// Gateway is the name of the Gateway object for the RayService to serve traffics during an IncrementalUpgrade.
```

Comment:
Add a comment saying that the RayServiceIncrementalUpgrade feature gate is required